src/components/Footer.js

```diff
@@ -1,115 +1,47 @@
-import React from 'react'
-import { Link } from 'gatsby'
+import React from 'react';
+import { Link } from 'gatsby';
 
-import logo from '../img/logo.svg'
-import facebook from '../img/social/facebook.svg'
-import instagram from '../img/social/instagram.svg'
-import twitter from '../img/social/twitter.svg'
-import vimeo from '../img/social/vimeo.svg'
+import logo from '../img/logo.svg';
+import facebook from '../img/social/facebook.svg';
+import instagram from '../img/social/instagram.svg';
+import twitter from '../img/social/twitter.svg';
+import vimeo from '../img/social/vimeo.svg';
 
 const Footer = class extends React.Component {
   render() {
     return (
-      <footer className="footer has-background-black has-text-white-ter">
-        <div className="content has-text-centered">
-          <img
-            src={logo}
-            alt="Kaldi"
-            style={{ width: '14em', height: '10em' }}
-          />
+      <footer className="footer">
+        <div className="footer-social-media">
+          <h3 className="footer-title">Siga-nos nas redes sociais</h3>
+          <ul className="footer-sm-icons">
+            <li>
+              <a href="" target="_blank">
+                <img alt="Logo Facebook" src="~/static/img/facebook.svg" />
```

Comment:
Path aqui da imagem ficou quebrado: 
facebook.svg:1 Failed to load resource: the server responded with a status of 404 ()